packages/SwingSet/test/test-xsnap-store.js

```diff
@@ -130,6 +130,14 @@ test('create SES worker, save, restore, resume', async t => {
  * They are also sensitive to the XS code itself.
  */
 test('XS + SES snapshots are long-term deterministic', async t => {
+  /* global globalThis */
+  const ENDO_BRANCH = globalThis.process?.env?.ENDO_BRANCH;
+  if (ENDO_BRANCH && ENDO_BRANCH !== 'NOPE') {
```

Comment:
Wondering if instead we shouldn't run a `yarn test --update-snapshots` (and ignore the clobbered snapshot artifacts in the porcelain check)

Reply:
I don't want to make an assumption that this is the only SwingSet test using `t.snapshot`.  This approach seems less brittle.

Reply:
The way I see it, when we add a test that breaks under these endo changes, either we update the test, or we update the CI job to handle the test. The main problem is if there are golden test impacted by endo changes that do not use snapshots.

The CI config here could simply add `--update-snapshot` to all `yarn test` invocations if we're not worried about other snapshot based tests diverging in more significant ways.